pkg/generators/openapi_test.go

```diff
@@ -115,8 +115,8 @@ type Blah struct {
 	// +patchStrategy=ps
 	// +patchMergeKey=pmk
 	WithStructTagExtension string `+"`"+`patchStrategy:"ps" patchMergeKey:"pmk"`+"`"+`
-	// a member with a list attribute
-	// +listAttribute=la
+	// a member with a list type
+	// +listType=la
```

Comment:
Please use valid data in tests, since people look at them as examples.

Reply:
Also why is a list attribute put on a string? Is this testing that it emits a warning/error?

Reply:
Fixed. Used valid listType and put this extension on a list. The old tests are not on a list. Those old tests will be fixed when "gofmt" makes the output more readable in a future PR.